x/ethbridge/keeper/msg_server_test.go

```diff
@@ -0,0 +1,92 @@
+package keeper_test
+
+import (
+	keeper2 "github.com/Sifchain/sifnode/x/ethbridge/keeper"
+	"github.com/Sifchain/sifnode/x/ethbridge/test"
+	"github.com/Sifchain/sifnode/x/ethbridge/types"
+	types2 "github.com/Sifchain/sifnode/x/tokenregistry/types"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/stretchr/testify/require"
+	"testing"
+)
+
+func TestMsgServer_Lock(t *testing.T) {
+	ctx, app := test.CreateSimulatorApp(false)
+	receiverCoins := app.BankKeeper.GetAllBalances(ctx, cosmosReceivers[0])
```

Comment:
Why do we need this and the next line? Is it because we are reusing cosmosReceivers[0]? 
In this case, why do we have cosmosReceivers, instead of generating an address through test_helper::CreateTestAddrs inside the unit test?

Reply:
We don't , there were just remenants from the original testcase I used as a template , which does a balance check at the end .